### PR TITLE
Build and deploy pages on CI

### LINKED ISSUE #13: Set up automatic deployment of master branch
We should remove the need for manual deployments of the documentation. Instead we should automatically deploy the master branch if new changes are pushed.

### PR DESCRIPTION
**Pull request by geigerzaehler**
Fixes #13

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,7 +9,7 @@ node_modules
 .eslintcache
 
 yarn-error.log
-build
+/build
```
Comment (NunoAlexandre) [inline .gitignore:12]: will this be relative to the project?

File: README.md [hunk 2]
```diff
@@ -26,8 +26,15 @@ This command generates static content into the `build` directory and can be serv
 
 ### Deployment
 
+The master branch of this repo is automatically deployed to
+`registry.radicle.xyz` by [Buildkite][buildkite-pipeline] via Github Pages.
+
+You can also manually deploy the site.
```
Comment (NunoAlexandre) [inline README.md]:
```suggestion
You can also manually deploy the site by running:
```

**Review by NunoAlexandre — COMMENTED** [review]
Looks good but why do we need Docker? :thinking:

**Comment by geigerzaehler** [comment]
> Looks good but why do we need Docker? thinking

Because our buildkite agents only support running builds with docker.

File: .gitignore (hunk 1 above)
Comment (geigerzaehler) [reply .gitignore:12]: [Yes]. Otherwise `ci/build` would be ignored.

Comment (NunoAlexandre) [reply .gitignore:12]: Ah I see, good catch.

**Comment by geigerzaehler** [comment]
@NunoAlexandre could you take another look?

**Review by NunoAlexandre — APPROVED** [review]
🛳

**Pull request merged**